.circleci/base_config.yml

```diff
@@ -261,6 +261,40 @@ jobs:
                   - 3rdParty/iresearch/tests/resources
                   - 3rdParty/rta-makedata
 
+  compile-mac:
+    parameters:
+      preset:
+        type: string
+      build-tests:
+        type: boolean
+      publish-artifacts:
+        type: boolean
+        default: true
+    macos:
+      xcode: 14.2.0
+    resource_class: macos.m1.medium.gen1
+    steps:
+      - checkout-arangodb:
+          with-submodules: true
+          destination: "~/project"
+      - run:
+          name: "Install"
+          command: |
+            brew install cmake openssl@3 llvm@14
```

Comment:
Doesn't match exact OpenSSL version we use through `.\VERSIONS`.

Reply:
Its using 3.1.3, which is the same as ours.
But I get your comment that its not directly specified, will look at a solution.
All have to be build on each PR, as we cant have images like with windows and Linux

Reply:
OpenSSL version taken from VERSIONS is used on build